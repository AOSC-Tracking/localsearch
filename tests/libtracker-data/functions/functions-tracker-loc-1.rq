PREFIX ex: <http://example/>
PREFIX ns: <http://www.w3.org/2005/xpath-functions#>

SELECT ?location tracker:cartesian-distance(?lat1,?lat2,?lon1,?lon2) tracker:haversine-distance(?lat1,?lat2,?lon1,?lon2)
{ ?_x a ex:Location ;
      ex:name ?location ;
      ex:latitude ?lat1 ;
      ex:longitude ?lon1 .
 ex:x ex:latitude ?lat2 ;
      ex:longitude ?lon2 .
}
