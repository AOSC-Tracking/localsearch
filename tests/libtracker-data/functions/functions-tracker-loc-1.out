"Helsinki"	"0.0"	"0.0"
"Tampere"	"161905.358015775"	"160759.509081122"
"London"	"1898317.54736874"	"1821325.85595572"
"Tuvalu"	"14017010.3444867"	"13884463.8434573"
